src/api/veilarbpersonflatefs.ts

```diff
@@ -6,13 +6,15 @@ export const STOPPE_VEDTAKSUTSENDING_TOGGLE = 'veilarbvedtaksstottefs.stoppevedt
 export const PILOT_TOGGLE = 'pto.vedtaksstotte.pilot';
 export const SKRU_AV_POLLING_UTKAST = 'veilarbvedtaksstottefs.skru_av_polling_utkast';
 export const SKRU_AV_POLLING_DIALOG = 'veilarbvedtaksstottefs.skru_av_polling_dialog';
+export const PERSONALIA_DATA_FRA_PDL = 'veilarbmaofs.personalia.pdl.persondata';
```

Comment:
Tenker det er tryggest å lage en egen toggle for bruk av målform fra PDL i vedtaksstøtte.

F.eks `veilarbvedtaksstottefs.hent_malform_fra_pdl`